ehrapy/anndata/_constants.py

```diff
@@ -6,3 +6,13 @@
 NUMERIC_TAG = "numeric"
 CATEGORICAL_TAG = "categorical"
 DATE_TAG = "date"
+
+NORM_NAMES = {
+    "StandardScaler" : "scale",
+    "MinMaxScaler" : "minmax", 
+    "MaxAbsScaler" : "maxabs",
+    "PowerTransformer" : "quantile",
+    "QuantileTransformer" : "power",
+    "RobustScaler" : "robust_scale",
+    "log_norm" : "log"
+}
```

Comment:
these constants have been distributed across `_normalization.py` before

Reply:
Hmm but these are not `anndata` constants, they are `normalization` constants. If we had a global constants module at the root level of ehrapy, I'd get it...

Reply:
We can also make such a global constants module at the root level of ehrapy - so far, I've taken the anndata constants to be this (since its constants "added" to anndata which in some cases can tune functionality, although for the normalization this is not yet the case)

Reply:
moved back now